src/compute-pr-actions.ts

```diff
@@ -352,7 +352,7 @@ export function process(prInfo: PrInfo | BotEnsureRemovedFromProject | BotNoPack
         }
         // Ping stale reviewers if any
         if (info.staleReviews.length > 0) {
-            const oid = info.staleReviews.slice().sort((l, r) => l.date.getTime() - r.date.getTime())[0].abbrOid;
+            const oid = info.staleReviews.reduce((l, r) => r.date < l.date ? r : l).abbrOid;

```

Comment:
(Isn't `earliestDate` better here?)

Reply:
Definitely, if it supported a callback to compare on a property (`.date`). I added the callback and renamed `earliestDate()` -> `min()` and `latestDate()` -> `max()`, since they aren't date specific?

Reply:
Yeah, that makes sense.